src/app/legacy/components/RadioSchedule/utilities/index.js

```diff
@@ -1,31 +1,31 @@
 import { formatDuration } from '#psammead/psammead-timestamp-container/src/utilities';
 import {
-  C_KINGFISHER,
-  C_POSTBOX,
-  C_EBON,
-  C_SHADOW,
-  C_WHITE,
-  C_METAL,
-} from '#psammead/psammead-styles/src/colours';
+  KINGFISHER,
```

Comment:
Would we consider refactoring this in the future so it doesn't rely on imports? It seems ok to me to leave it for the time being.